packages/perseus-core/src/data-schema.ts

```diff
@@ -144,7 +144,6 @@ export interface PerseusWidgetTypes {
     matcher: MatcherWidget;
     matrix: MatrixWidget;
     measurer: MeasurerWidget;
-    "mock-widget": MockWidget;
```

Comment:
Removing here as we don't want the MockWidget to be part of the public set of widget types. It is an internal, test-only widget.